src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -1336,7 +1338,13 @@ const formConfig = {
           CustomPage: ApplicantMedicareStatusPage,
           CustomPageReview: ApplicantMedicareStatusReviewPage,
           schema: applicantListSchema([], {
-            applicantMedicareStatus: { type: 'string' },
+            applicantMedicareStatus: {
+              type: 'object',
+              properties: {
+                eligibility: { type: 'string' },
+                otherIneligible: { type: 'string' },
+              },
+            },
```

Comment:
Updating so it reflects the reality of the custom page's input. Also updated pages that check this via `depends` so they drill down the right property.